Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -35,9 +35,9 @@ namespace AudioCommon
 		std::string backend = SConfig::GetInstance().sBackend;
 		if (backend == BACKEND_OPENAL && OpenALStream::isValid())
 			g_sound_stream = new OpenALStream();
-		else if (backend == BACKEND_NULLSOUND && NullSound::isValid())
+		else if (backend.c_str() == BACKEND_NULLSOUND && NullSound::isValid())
```

Comment:
While it was not necessary, I thought that it was a little more correct, since to me it otherwise reads "compare the pointer to a c string to a pointer to a c++ string" versus their c string values.  I will revert that if you disagree.